include/uxr/agent/transport/udp/UDPServerLinux.hpp

```diff
@@ -35,7 +35,7 @@ class UDPServer : public UDPServerBase
     ~UDPServer() = default;
 
 private:
-    virtual bool init() override;
+    virtual bool init(bool discovery_enabled) override;
```

Comment:
To keep consistency remove `virtual` from derived class overridden methods. If the derived class should not have more derived, consider `final` keyword.

Inconsistency between SerialServers and TCP and UDP servers.